packages/nocodb/src/services/utils.service.ts

```
@@ -471,4 +479,106 @@
+      );
+    } catch (e) {
+      this.logger.error(e?.message, e);
+      return [];
+    }
+
+    // The feed includes the attachments, which has the presigned URL
+    // So the cache should match the presigned URL cache
+    await NocoCache.setExpiring(
+      cacheKey,
+      JSON.stringify(response.data, getCircularReplacer),
+      isNaN(parseInt(process.env.NC_ATTACHMENT_EXPIRE_SECONDS))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)